cpp/src/IceIAP/Transceiver.h

```diff
@@ -18,7 +18,10 @@
 namespace IceObjC
 {
 
-class iAPTransceiver final : public IceInternal::Transceiver, public IceInternal::StreamNativeInfo
+class iAPTransceiver final :
+    public IceInternal::Transceiver,
+    public IceInternal::StreamNativeInfo,
+    public std::enable_shared_from_this<iAPTransceiver>
```

Comment:
Do we also need to call the `enable_shared_form_this` constructor in the `iAPTransceiver` constructor?

Reply:
nvm, I'm not sure why I thought it was necessary.